services/common/src/components/projectSummary/DocumentUpload.tsx

```diff
@@ -67,9 +68,10 @@ const RenderOldDocuments = ({
 
 interface DocumentUploadProps {
   docFieldsDisabled: boolean;
+  fieldsDisabled: boolean;
```

Comment:
Yeah.... I'm questioning this one. 
- the purpose of it seems to be controlling _deletion_, so if it's separate from docFieldsDisabled I think it should be named explicitly like "canDelete" or "deleteDisabled" or whatever (keyword: delete)
- tying it to the authorization fields in the parent component and then a different value in the test is kind of confusing. 
- but it's getting late in the day and maybe I'm overcomplicating things

Reply:
<img width="858" alt="image" src="https://github.com/user-attachments/assets/d8e8c0d4-aa9a-4477-b2ed-b8c7e61af289" />

Happy to change the field name but I was just keeping it the same as all the other tabs

<img width="885" alt="image" src="https://github.com/user-attachments/assets/7de877e9-a800-4f6d-9c07-2ca0c82317d9" />

Reply:
Although it looks like I did use authFieldsDisabled instead of fieldsDisabled.... oops